app/ante/eth_set_pubkey_test.go

```diff
@@ -153,19 +153,6 @@ func (suite *AnteTestSuite) TestEthSetPubkeyHandler() {
 				suite.Require().Error(err)
 			}
 			suite.Require().True(tc.correctPubKey(pubKey), "PubKey type incorrect after AnteHandler")
-
-			// Now check the old antehandler against the same Tx, see if it would create a pubkey
-			ctx, _ = suite.ctx.CacheContext()
```

Comment:
why was this removed?

Reply:
This check was to compare an older version of the antehandler that lives in the Canto repo to a newer one we were deploying. We no longer have that old antehandler in place so testing against it is no longer relevant and I believe it was causing me problems but I can't remember which problems at this point.